content/enterprise_influxdb/v1.9/about-the-project/release-notes-changelog.md

```diff
@@ -1,36 +1,36 @@
 ---
 title: InfluxDB Enterprise 1.9 release notes
 description: >
-  Important changes and and what's new in each version InfluxDB Enterprise.
+  Important changes and what's new in each version InfluxDB Enterprise.
 menu:
   enterprise_influxdb_1_9_ref:
     name: Release notes
     weight: 10
     parent: About the project
 ---
 
-v1.9.0 [unreleased]
--------------------
+## v1.9.0 [TK]
 
 ### Features
 - Support user-defined *node labels*.
   Node labels allows cluster operators to assign arbitrary key-value pairs to meta and data nodes in a cluster.
   For instance, an operator might want to label nodes with the availability zone in which they're located.
-- Improve diagnostics for license problems.
-- Ingress metrics now work.
+- Improve diagnostics for license problems. Add [license expiration date](/enterprise_influxdb/v1.9/features/clustering-features/#entitlements) to `debug/vars` metrics.
+- Improved ingress metrics allow tracking points written per-measurement and per-login.
 - Support authorization for Kapacitor via LDAP.
-- Flux query controls. <!-- #3373: feat:  -->
-- Support for pushed-down window aggregates. <!-- #3372: feat:  -->
-
-
-### Bugfixes
-- Show databases now checks read write permissions. <!-- https://github.com/influxdata/plutonium/pull/3341 -->
-- Anti-entrpoy: Update `tsm1.BlockCount()` call to match signature <!-- https://github.com/influxdata/plutonium/pull/3152 -->
-- Remove extraneous nil check from points writer. <!-- https://github.com/influxdata/plutonium/pull/3141 -->
-- Restore: Ensure a newline is printed after a successful copy <!-- https://github.com/influxdata/plutonium/pull/3137 -->
-- Remove extraneous nil check <!-- #3141 -->
-- Make 'entropy show' expiry times consistent with 'show-shards' <!-- #3355: fix:  -->
+- Allow [configuring Flux query resource usage](enterprise_influxdb/v1.9/administration/config-data-nodes/#flux-controller) (concurrency, memory, etc.).
+- Upgrade to Flux v0.113.0. This includes:
```

Comment:
Will, typically we link to the Flux release notes....let's double-check these release notes--looks like there may be some discrepancy between what's listed here and what's actually in the release notes?